### PR TITLE
using get_bin_path() on atomic modules

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
These modules were calling `run_command()` without submitting the command through `get_bin_path()`.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/cloud/atomic/atomic_container.py
plugins/modules/cloud/atomic/atomic_host.py
plugins/modules/cloud/atomic/atomic_image.py

### CODE REVIEW

**Review by krsacme — APPROVED** [review]

**Comment by krsacme** [comment]
LGTM

**Comment by felixfontein** [comment]
Can you add a changelog fragment?

**Review by giuseppe — APPROVED** [review]
LGTM

File: changelogs/fragments/2144-atomic_get_bin_path.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - atomic_container - using `get_bin_path()` before calling `run_command()` (https://github.com/ansible-collections/community.general/pull/2144).
+  - atomic_host - using `get_bin_path()` before calling `run_command()` (https://github.com/ansible-collections/community.general/pull/2144).
+  - atomic_image - using `get_bin_path()` before calling `run_command()` (https://github.com/ansible-collections/community.general/pull/2144).
```
Comment (felixfontein) [inline changelogs/fragments/2144-atomic_get_bin_path.yml:4]:
```suggestion
  - atomic_container - using ``get_bin_path()`` before calling ``run_command()`` (https://github.com/ansible-collections/community.general/pull/2144).
  - atomic_host - using ``get_bin_path()`` before calling ``run_command()`` (https://github.com/ansible-collections/community.general/pull/2144).
  - atomic_image - using ``get_bin_path()`` before calling ``run_command()`` (https://github.com/ansible-collections/community.general/pull/2144).
```

**Review by felixfontein — COMMENTED** [review]
Besides that, looks good.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for the improvements!
@krsacme @giuseppe thanks for reviewing!

**Pull request merged**